### ISSUE TITLE
Empty crockpot interacting with an empty bowl cause an exception

### ISSUE THREAD

**Issue by darnuria** [anegostudios/VintageStory-Issues#534]
**Game Version:** v1.12.14 (Stable)
**Platform:** Linux, Ubuntu 18.04.4 LTS
**Modded:** Yes only VSHUD Novocain mod
**Processor:** Intel(R) Core(TM) i5-5300U CPU @ 2.30GHz
**MonoVersion:** C# Framework: Mono 6.8.0.105 (tarball Tue Feb 4 21:20:20 UTC 2020)
**Gamemode**: Survival solo get a not fatal exception, multiplayer crash? (need test)
Not reproducible in creative

Bug found by LadySnapdragonToday on discord 2020-05-19 at 5:49 PM
reproduced by me ~18:00 same day.

### Description

Interacting with an empty bowl with an empty crockpot trigger an exception. In solo it's not fatal in multiplayer it cause a crash.

The bowl is erased in the process.

### How to reproduce

0. Place a bowl on an empty place
1. Interact with the bowl with an empty crockpot
Result: Whof the bowl disappears exception triggered

Bug reproducible on Chrometech at 2020-05-19 18:15.

### Screenshot

The setup:

![Screenshot from 2020-05-19 17-59-39](https://user-images.githubusercontent.com/2827553/82350658-eec5c180-99fb-11ea-81d0-409981676a66.png)

After interacting bowl vanished:
![Screenshot from 2020-05-19 17-59-42](https://user-images.githubusercontent.com/2827553/82350706-fd13dd80-99fb-11ea-8a09-47de63915eb7.png)

Outcome in multiplayers
![Screenshot from 2020-05-19 18-15-19](https://user-images.githubusercontent.com/2827553/82351397-d2765480-99fc-11ea-8b04-766ae96e14f6.png)

#### Solo

```
18:01:06 [Server Chat] Darnuria: <strong>Darnuria:</strong> interacting with bowl
18:01:06 [Client Chat] <strong>Darnuria:</strong> interacting with bowl @ 0
18:01:07 [Server Error] System.NullReferenceException: Object reference not set to an instance of an object
  at Vintagestory.API.Common.AssetLocation.ResolveToDomainAndPath (System.String domainAndPath, System.String& domain, System.String& path) [0x00000] in <97f87400996549bcaa796080030a18a5>:0 
  at Vintagestory.API.Common.AssetLocation..ctor (System.String domainAndPath) [0x00006] in <97f87400996549bcaa796080030a18a5>:0 
  at Vintagestory.GameContent.BlockCrock.ServeIntoBowl (Vintagestory.API.Common.Block selectedBlock, Vintagestory.API.MathTools.BlockPos pos, Vintagestory.API.Common.ItemSlot potslot, Vintagestory.API.Common.IWorldAccessor world) [0x00156] in <e160795a538f4d649533661f023b3166>:0 
  at Vintagestory.GameContent.BlockCrock.OnHeldInteractStart (Vintagestory.API.Common.ItemSlot slot, Vintagestory.API.Common.EntityAgent byEntity, Vintagestory.API.Common.BlockSelection blockSel, Vintagestory.API.Common.EntitySelection entitySel, System.Boolean firstEvent, Vintagestory.API.Common.EnumHandHandling& handHandling) [0x00093] in <e160795a538f4d649533661f023b3166>:0 
  at Vintagestory.API.Common.CollectibleObject.OnHeldUseStart (Vintagestory.API.Common.ItemSlot slot, Vintagestory.API.Common.EntityAgent byEntity, Vintagestory.API.Common.BlockSelection blockSel, Vintagestory.API.Common.EntitySelection entitySel, Vintagestory.API.Common.EnumHandInteract useType, System.Boolean firstEvent, Vintagestory.API.Common.EnumHandHandling& handling) [0x00018] in <97f87400996549bcaa796080030a18a5>:0 
  at Vintagestory.Server.ServerSystemInventory.HandleHandInteraction (_zPEoTHWUNtCMZ1Y7jHl7xInordd packet, Vintagestory.Server.ConnectedClient client) [0x001da] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 
  at Vintagestory.Server.ServerMain.HandleClientPacket (Vintagestory.Server.ConnectedClient client, System.Byte[] data) [0x0005a] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 
  at Vintagestory.Server.ServerMain.ProcessNetMessage (Vintagestory.Common.NetIncomingMessage msg, Vintagestory.Common.NetServer mainSocket) [0x0014d] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 
```
#### Multiplayers

```
18:14:58 [Client Chat] <strong>Darnuria:</strong> I gonna crash (again) @ 0
18:15:07 [Client Chat] <strong>Darnuria:</strong> Hello all :D @ 0
18:15:16 [Client Notification] Disconnected by the server (An action you (or your client) did caused an unhandled exception)
18:15:16 [Client Notification] Destroying game session, waiting up to 200ms for client threads to exit
18:15:16 [Client Notification] Found 8 base assets in category lang
18:15:16 [Client Notification] Found 0 base assets in category journal
18:15:16 [Client Notification] Found 0 base assets in category patches
18:15:16 [Client Notification] Found 16 base assets in category config
18:15:16 [Client Notification] Found 0 base assets in category worldproperties
18:15:16 [Client Error] Exception thrown during OnTesselation() of block entity Vintagestory.GameContent.BlockEntityFirepit@347539/110/295119: System.NullReferenceException: Object reference not set to an instance of an object
  at Vintagestory.Client.NoObf.ShapeTesselator.TesselateShape (System.String typeForLogging, Vintagestory.API.Common.Shape shapeBase, Vintagestory.API.Client.MeshData& modeldata, Vintagestory.API.Client.ITexPositionSource texSource, Vintagestory.API.MathTools.Vec3f wholeMeshRotation, Vintagestory.API.MathTools.Vec3f wholeMeshOffset, System.Int32 generalGlowLevel, System.Int32 generalTintIndex, System.Nullable`1[T] quantityElements, System.Boolean withElementIndices, System.String[] selectiveElements) [0x000a4] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 
  at Vintagestory.Client.NoObf.ShapeTesselator.TesselateShape (System.String typeForLogging, Vintagestory.API.Common.Shape shapeBase, Vintagestory.API.Client.MeshData& modeldata, Vintagestory.API.Client.ITexPositionSource texSource, Vintagestory.API.MathTools.Vec3f wholeMeshRotation, System.Int32 generalGlowLevel, System.Int32 generalTintIndex, System.Nullable`1[T] quantityElements, System.String[] selectiveElements) [0x00000] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 
  at Vintagestory.Client.NoObf.ShapeTesselator.TesselateShape (Vintagestory.API.Common.Block textureSourceBlock, Vintagestory.API.Common.Shape shape, Vintagestory.API.Client.MeshData& modeldata, Vintagestory.API.MathTools.Vec3f rotation, System.Nullable`1[T] quantityElements, System.String[] selectiveElements) [0x0001e] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 
  at Vintagestory.GameContent.BlockEntityFirepit.getOrCreateMesh (System.String burnstate, System.String contentstate) [0x000b5] in <e160795a538f4d649533661f023b3166>:0 
  at Vintagestory.GameContent.BlockEntityFirepit.OnTesselation (Vintagestory.API.Client.ITerrainMeshPool mesher, Vintagestory.API.Client.ITesselatorAPI tesselator) [0x000a1] in <e160795a538f4d649533661f023b3166>:0 
  at Vintagestory.Client.NoObf.JsonTesselator.Tesselate (Vintagestory.Client.NoObf.TCTCache vars) [0x00145] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 . Block will probably not be rendered as intended.
18:15:16 [Client Notification] Found 31 base assets in category sounds
18:15:16 [Client Notification] Found 2 base assets in category shapes
18:15:16 [Client Notification] Found 72 base assets in category shaders
18:15:16 [Client Notification] Found 13 base assets in category shaderincludes
18:15:16 [Client Notification] Found 58 base assets in category textures
18:15:16 [Client Notification] Found 3 base assets in category music
18:15:16 [Client Notification] Found 0 base assets in category dialog
18:15:16 [Client Error] Exception thrown during OnTesselation() of block entity Vintagestory.GameContent.BlockEntityFirepit@347547/110/295129: System.NullReferenceException: Object reference not set to an instance of an object
  at Vintagestory.Client.NoObf.ShapeTesselator.TesselateShape (System.String typeForLogging, Vintagestory.API.Common.Shape shapeBase, Vintagestory.API.Client.MeshData& modeldata, Vintagestory.API.Client.ITexPositionSource texSource, Vintagestory.API.MathTools.Vec3f wholeMeshRotation, Vintagestory.API.MathTools.Vec3f wholeMeshOffset, System.Int32 generalGlowLevel, System.Int32 generalTintIndex, System.Nullable`1[T] quantityElements, System.Boolean withElementIndices, System.String[] selectiveElements) [0x000a4] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 
  at Vintagestory.Client.NoObf.ShapeTesselator.TesselateShape (System.String typeForLogging, Vintagestory.API.Common.Shape shapeBase, Vintagestory.API.Client.MeshData& modeldata, Vintagestory.API.Client.ITexPositionSource texSource, Vintagestory.API.MathTools.Vec3f wholeMeshRotation, System.Int32 generalGlowLevel, System.Int32 generalTintIndex, System.Nullable`1[T] quantityElements, System.String[] selectiveElements) [0x00000] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 
  at Vintagestory.Client.NoObf.ShapeTesselator.TesselateShape (Vintagestory.API.Common.Block textureSourceBlock, Vintagestory.API.Common.Shape shape, Vintagestory.API.Client.MeshData& modeldata, Vintagestory.API.MathTools.Vec3f rotation, System.Nullable`1[T] quantityElements, System.String[] selectiveElements) [0x0001e] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 
  at Vintagestory.GameContent.BlockEntityFirepit.getOrCreateMesh (System.String burnstate, System.String contentstate) [0x000b5] in <e160795a538f4d649533661f023b3166>:0 
  at Vintagestory.GameContent.BlockEntityFirepit.OnTesselation (Vintagestory.API.Client.ITerrainMeshPool mesher, Vintagestory.API.Client.ITesselatorAPI tesselator) [0x000a1] in <e160795a538f4d649533661f023b3166>:0 
  at Vintagestory.Client.NoObf.JsonTesselator.Tesselate (Vintagestory.Client.NoObf.TCTCache vars) [0x00145] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 . Block will probably not be rendered as intended.
18:15:16 [Client Error] Exception thrown during OnTesselation() of block entity Vintagestory.GameContent.BlockEntityFirepit@347547/110/295130: System.NullReferenceException: Object reference not set to an instance of an object
  at Vintagestory.Client.NoObf.ShapeTesselator.TesselateShape (System.String typeForLogging, Vintagestory.API.Common.Shape shapeBase, Vintagestory.API.Client.MeshData& modeldata, Vintagestory.API.Client.ITexPositionSource texSource, Vintagestory.API.MathTools.Vec3f wholeMeshRotation, Vintagestory.API.MathTools.Vec3f wholeMeshOffset, System.Int32 generalGlowLevel, System.Int32 generalTintIndex, System.Nullable`1[T] quantityElements, System.Boolean withElementIndices, System.String[] selectiveElements) [0x000a4] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 
  at Vintagestory.Client.NoObf.ShapeTesselator.TesselateShape (System.String typeForLogging, Vintagestory.API.Common.Shape shapeBase, Vintagestory.API.Client.MeshData& modeldata, Vintagestory.API.Client.ITexPositionSource texSource, Vintagestory.API.MathTools.Vec3f wholeMeshRotation, System.Int32 generalGlowLevel, System.Int32 generalTintIndex, System.Nullable`1[T] quantityElements, System.String[] selectiveElements) [0x00000] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 
  at Vintagestory.Client.NoObf.ShapeTesselator.TesselateShape (Vintagestory.API.Common.Block textureSourceBlock, Vintagestory.API.Common.Shape shape, Vintagestory.API.Client.MeshData& modeldata, Vintagestory.API.MathTools.Vec3f rotation, System.Nullable`1[T] quantityElements, System.String[] selectiveElements) [0x0001e] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 
  at Vintagestory.GameContent.BlockEntityFirepit.getOrCreateMesh (System.String burnstate, System.String contentstate) [0x000b5] in <e160795a538f4d649533661f023b3166>:0 
  at Vintagestory.GameContent.BlockEntityFirepit.OnTesselation (Vintagestory.API.Client.ITerrainMeshPool mesher, Vintagestory.API.Client.ITesselatorAPI tesselator) [0x000a1] in <e160795a538f4d649533661f023b3166>:0 
  at Vintagestory.Client.NoObf.JsonTesselator.Tesselate (Vintagestory.Client.NoObf.TCTCache vars) [0x00145] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 . Block will probably not be rendered as intended.
18:15:16 [Client Error] Exception thrown during OnTesselation() of block entity Vintagestory.GameContent.BlockEntityFirepit@347548/110/295129: System.NullReferenceException: Object reference not set to an instance of an object
  at Vintagestory.Client.NoObf.ShapeTesselator.TesselateShape (System.String typeForLogging, Vintagestory.API.Common.Shape shapeBase, Vintagestory.API.Client.MeshData& modeldata, Vintagestory.API.Client.ITexPositionSource texSource, Vintagestory.API.MathTools.Vec3f wholeMeshRotation, Vintagestory.API.MathTools.Vec3f wholeMeshOffset, System.Int32 generalGlowLevel, System.Int32 generalTintIndex, System.Nullable`1[T] quantityElements, System.Boolean withElementIndices, System.String[] selectiveElements) [0x000a4] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 
  at Vintagestory.Client.NoObf.ShapeTesselator.TesselateShape (System.String typeForLogging, Vintagestory.API.Common.Shape shapeBase, Vintagestory.API.Client.MeshData& modeldata, Vintagestory.API.Client.ITexPositionSource texSource, Vintagestory.API.MathTools.Vec3f wholeMeshRotation, System.Int32 generalGlowLevel, System.Int32 generalTintIndex, System.Nullable`1[T] quantityElements, System.String[] selectiveElements) [0x00000] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 
  at Vintagestory.Client.NoObf.ShapeTesselator.TesselateShape (Vintagestory.API.Common.Block textureSourceBlock, Vintagestory.API.Common.Shape shape, Vintagestory.API.Client.MeshData& modeldata, Vintagestory.API.MathTools.Vec3f rotation, System.Nullable`1[T] quantityElements, System.String[] selectiveElements) [0x0001e] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 
  at Vintagestory.GameContent.BlockEntityFirepit.getOrCreateMesh (System.String burnstate, System.String contentstate) [0x000b5] in <e160795a538f4d649533661f023b3166>:0 
  at Vintagestory.GameContent.BlockEntityFirepit.OnTesselation (Vintagestory.API.Client.ITerrainMeshPool mesher, Vintagestory.API.Client.ITesselatorAPI tesselator) [0x000a1] in <e160795a538f4d649533661f023b3166>:0 
  at Vintagestory.Client.NoObf.JsonTesselator.Tesselate (Vintagestory.Client.NoObf.TCTCache vars) [0x00145] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 . Block will probably not be rendered as intended.
18:15:16 [Client Error] Exception thrown during OnTesselation() of block entity Vintagestory.GameContent.BlockEntityFirepit@347548/110/295130: System.NullReferenceException: Object reference not set to an instance of an object
  at Vintagestory.Client.NoObf.ShapeTesselator.TesselateShape (System.String typeForLogging, Vintagestory.API.Common.Shape shapeBase, Vintagestory.API.Client.MeshData& modeldata, Vintagestory.API.Client.ITexPositionSource texSource, Vintagestory.API.MathTools.Vec3f wholeMeshRotation, Vintagestory.API.MathTools.Vec3f wholeMeshOffset, System.Int32 generalGlowLevel, System.Int32 generalTintIndex, System.Nullable`1[T] quantityElements, System.Boolean withElementIndices, System.String[] selectiveElements) [0x000a4] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 
  at Vintagestory.Client.NoObf.ShapeTesselator.TesselateShape (System.String typeForLogging, Vintagestory.API.Common.Shape shapeBase, Vintagestory.API.Client.MeshData& modeldata, Vintagestory.API.Client.ITexPositionSource texSource, Vintagestory.API.MathTools.Vec3f wholeMeshRotation, System.Int32 generalGlowLevel, System.Int32 generalTintIndex, System.Nullable`1[T] quantityElements, System.String[] selectiveElements) [0x00000] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 
  at Vintagestory.Client.NoObf.ShapeTesselator.TesselateShape (Vintagestory.API.Common.Block textureSourceBlock, Vintagestory.API.Common.Shape shape, Vintagestory.API.Client.MeshData& modeldata, Vintagestory.API.MathTools.Vec3f rotation, System.Nullable`1[T] quantityElements, System.String[] selectiveElements) [0x0001e] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 
  at Vintagestory.GameContent.BlockEntityFirepit.getOrCreateMesh (System.String burnstate, System.String contentstate) [0x000b5] in <e160795a538f4d649533661f023b3166>:0 
  at Vintagestory.GameContent.BlockEntityFirepit.OnTesselation (Vintagestory.API.Client.ITerrainMeshPool mesher, Vintagestory.API.Client.ITesselatorAPI tesselator) [0x000a1] in <e160795a538f4d649533661f023b3166>:0 
  at Vintagestory.Client.NoObf.JsonTesselator.Tesselate (Vintagestory.Client.NoObf.TCTCache vars) [0x00145] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 . Block will probably not be rendered as intended.
18:15:16 [Client Error] Exception thrown during OnTesselation() of block entity Vintagestory.GameContent.BlockEntityFirepit@347539/110/295119: System.NullReferenceException: Object reference not set to an instance of an object
  at Vintagestory.Client.NoObf.ShapeTesselator.TesselateShape (System.String typeForLogging, Vintagestory.API.Common.Shape shapeBase, Vintagestory.API.Client.MeshData& modeldata, Vintagestory.API.Client.ITexPositionSource texSource, Vintagestory.API.MathTools.Vec3f wholeMeshRotation, Vintagestory.API.MathTools.Vec3f wholeMeshOffset, System.Int32 generalGlowLevel, System.Int32 generalTintIndex, System.Nullable`1[T] quantityElements, System.Boolean withElementIndices, System.String[] selectiveElements) [0x000a4] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 
  at Vintagestory.Client.NoObf.ShapeTesselator.TesselateShape (System.String typeForLogging, Vintagestory.API.Common.Shape shapeBase, Vintagestory.API.Client.MeshData& modeldata, Vintagestory.API.Client.ITexPositionSource texSource, Vintagestory.API.MathTools.Vec3f wholeMeshRotation, System.Int32 generalGlowLevel, System.Int32 generalTintIndex, System.Nullable`1[T] quantityElements, System.String[] selectiveElements) [0x00000] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 
  at Vintagestory.Client.NoObf.ShapeTesselator.TesselateShape (Vintagestory.API.Common.Block textureSourceBlock, Vintagestory.API.Common.Shape shape, Vintagestory.API.Client.MeshData& modeldata, Vintagestory.API.MathTools.Vec3f rotation, System.Nullable`1[T] quantityElements, System.String[] selectiveElements) [0x0001e] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 
  at Vintagestory.GameContent.BlockEntityFirepit.getOrCreateMesh (System.String burnstate, System.String contentstate) [0x000b5] in <e160795a538f4d649533661f023b3166>:0 
  at Vintagestory.GameContent.BlockEntityFirepit.OnTesselation (Vintagestory.API.Client.ITerrainMeshPool mesher, Vintagestory.API.Client.ITesselatorAPI tesselator) [0x000a1] in <e160795a538f4d649533661f023b3166>:0 
  at Vintagestory.Client.NoObf.JsonTesselator.Tesselate (Vintagestory.Client.NoObf.TCTCache vars) [0x00145] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 . Block will probably not be rendered as intended.
18:15:16 [Client Error] Exception thrown during OnTesselation() of block entity Vintagestory.GameContent.BlockEntityFirepit@347547/110/295129: System.NullReferenceException: Object reference not set to an instance of an object
  at Vintagestory.Client.NoObf.ShapeTesselator.TesselateShape (System.String typeForLogging, Vintagestory.API.Common.Shape shapeBase, Vintagestory.API.Client.MeshData& modeldata, Vintagestory.API.Client.ITexPositionSource texSource, Vintagestory.API.MathTools.Vec3f wholeMeshRotation, Vintagestory.API.MathTools.Vec3f wholeMeshOffset, System.Int32 generalGlowLevel, System.Int32 generalTintIndex, System.Nullable`1[T] quantityElements, System.Boolean withElementIndices, System.String[] selectiveElements) [0x000a4] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 
  at Vintagestory.Client.NoObf.ShapeTesselator.TesselateShape (System.String typeForLogging, Vintagestory.API.Common.Shape shapeBase, Vintagestory.API.Client.MeshData& modeldata, Vintagestory.API.Client.ITexPositionSource texSource, Vintagestory.API.MathTools.Vec3f wholeMeshRotation, System.Int32 generalGlowLevel, System.Int32 generalTintIndex, System.Nullable`1[T] quantityElements, System.String[] selectiveElements) [0x00000] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 
  at Vintagestory.Client.NoObf.ShapeTesselator.TesselateShape (Vintagestory.API.Common.Block textureSourceBlock, Vintagestory.API.Common.Shape shape, Vintagestory.API.Client.MeshData& modeldata, Vintagestory.API.MathTools.Vec3f rotation, System.Nullable`1[T] quantityElements, System.String[] selectiveElements) [0x0001e] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 
  at Vintagestory.GameContent.BlockEntityFirepit.getOrCreateMesh (System.String burnstate, System.String contentstate) [0x000b5] in <e160795a538f4d649533661f023b3166>:0 
  at Vintagestory.GameContent.BlockEntityFirepit.OnTesselation (Vintagestory.API.Client.ITerrainMeshPool mesher, Vintagestory.API.Client.ITesselatorAPI tesselator) [0x000a1] in <e160795a538f4d649533661f023b3166>:0 
  at Vintagestory.Client.NoObf.JsonTesselator.Tesselate (Vintagestory.Client.NoObf.TCTCache vars) [0x00145] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 . Block will probably not be rendered as intended.
18:15:16 [Client Error] Exception thrown during OnTesselation() of block entity Vintagestory.GameContent.BlockEntityFirepit@347547/110/295130: System.NullReferenceException: Object reference not set to an instance of an object
  at Vintagestory.Client.NoObf.ShapeTesselator.TesselateShape (System.String typeForLogging, Vintagestory.API.Common.Shape shapeBase, Vintagestory.API.Client.MeshData& modeldata, Vintagestory.API.Client.ITexPositionSource texSource, Vintagestory.API.MathTools.Vec3f wholeMeshRotation, Vintagestory.API.MathTools.Vec3f wholeMeshOffset, System.Int32 generalGlowLevel, System.Int32 generalTintIndex, System.Nullable`1[T] quantityElements, System.Boolean withElementIndices, System.String[] selectiveElements) [0x000a4] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 
  at Vintagestory.Client.NoObf.ShapeTesselator.TesselateShape (System.String typeForLogging, Vintagestory.API.Common.Shape shapeBase, Vintagestory.API.Client.MeshData& modeldata, Vintagestory.API.Client.ITexPositionSource texSource, Vintagestory.API.MathTools.Vec3f wholeMeshRotation, System.Int32 generalGlowLevel, System.Int32 generalTintIndex, System.Nullable`1[T] quantityElements, System.String[] selectiveElements) [0x00000] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 
  at Vintagestory.Client.NoObf.ShapeTesselator.TesselateShape (Vintagestory.API.Common.Block textureSourceBlock, Vintagestory.API.Common.Shape shape, Vintagestory.API.Client.MeshData& modeldata, Vintagestory.API.MathTools.Vec3f rotation, System.Nullable`1[T] quantityElements, System.String[] selectiveElements) [0x0001e] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 
  at Vintagestory.GameContent.BlockEntityFirepit.getOrCreateMesh (System.String burnstate, System.String contentstate) [0x000b5] in <e160795a538f4d649533661f023b3166>:0 
  at Vintagestory.GameContent.BlockEntityFirepit.OnTesselation (Vintagestory.API.Client.ITerrainMeshPool mesher, Vintagestory.API.Client.ITesselatorAPI tesselator) [0x000a1] in <e160795a538f4d649533661f023b3166>:0 
  at Vintagestory.Client.NoObf.JsonTesselator.Tesselate (Vintagestory.Client.NoObf.TCTCache vars) [0x00145] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 . Block will probably not be rendered as intended.
18:15:16 [Client Error] Exception thrown during OnTesselation() of block entity Vintagestory.GameContent.BlockEntityFirepit@347548/110/295129: System.NullReferenceException: Object reference not set to an instance of an object
  at Vintagestory.Client.NoObf.ShapeTesselator.TesselateShape (System.String typeForLogging, Vintagestory.API.Common.Shape shapeBase, Vintagestory.API.Client.MeshData& modeldata, Vintagestory.API.Client.ITexPositionSource texSource, Vintagestory.API.MathTools.Vec3f wholeMeshRotation, Vintagestory.API.MathTools.Vec3f wholeMeshOffset, System.Int32 generalGlowLevel, System.Int32 generalTintIndex, System.Nullable`1[T] quantityElements, System.Boolean withElementIndices, System.String[] selectiveElements) [0x000a4] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 
  at Vintagestory.Client.NoObf.ShapeTesselator.TesselateShape (System.String typeForLogging, Vintagestory.API.Common.Shape shapeBase, Vintagestory.API.Client.MeshData& modeldata, Vintagestory.API.Client.ITexPositionSource texSource, Vintagestory.API.MathTools.Vec3f wholeMeshRotation, System.Int32 generalGlowLevel, System.Int32 generalTintIndex, System.Nullable`1[T] quantityElements, System.String[] selectiveElements) [0x00000] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 
  at Vintagestory.Client.NoObf.ShapeTesselator.TesselateShape (Vintagestory.API.Common.Block textureSourceBlock, Vintagestory.API.Common.Shape shape, Vintagestory.API.Client.MeshData& modeldata, Vintagestory.API.MathTools.Vec3f rotation, System.Nullable`1[T] quantityElements, System.String[] selectiveElements) [0x0001e] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 
  at Vintagestory.GameContent.BlockEntityFirepit.getOrCreateMesh (System.String burnstate, System.String contentstate) [0x000b5] in <e160795a538f4d649533661f023b3166>:0 
  at Vintagestory.GameContent.BlockEntityFirepit.OnTesselation (Vintagestory.API.Client.ITerrainMeshPool mesher, Vintagestory.API.Client.ITesselatorAPI tesselator) [0x000a1] in <e160795a538f4d649533661f023b3166>:0 
  at Vintagestory.Client.NoObf.JsonTesselator.Tesselate (Vintagestory.Client.NoObf.TCTCache vars) [0x00145] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 . Block will probably not be rendered as intended.
18:15:16 [Client Error] Exception thrown during OnTesselation() of block entity Vintagestory.GameContent.BlockEntityFirepit@347548/110/295130: System.NullReferenceException: Object reference not set to an instance of an object
  at Vintagestory.Client.NoObf.ShapeTesselator.TesselateShape (System.String typeForLogging, Vintagestory.API.Common.Shape shapeBase, Vintagestory.API.Client.MeshData& modeldata, Vintagestory.API.Client.ITexPositionSource texSource, Vintagestory.API.MathTools.Vec3f wholeMeshRotation, Vintagestory.API.MathTools.Vec3f wholeMeshOffset, System.Int32 generalGlowLevel, System.Int32 generalTintIndex, System.Nullable`1[T] quantityElements, System.Boolean withElementIndices, System.String[] selectiveElements) [0x000a4] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 
  at Vintagestory.Client.NoObf.ShapeTesselator.TesselateShape (System.String typeForLogging, Vintagestory.API.Common.Shape shapeBase, Vintagestory.API.Client.MeshData& modeldata, Vintagestory.API.Client.ITexPositionSource texSource, Vintagestory.API.MathTools.Vec3f wholeMeshRotation, System.Int32 generalGlowLevel, System.Int32 generalTintIndex, System.Nullable`1[T] quantityElements, System.String[] selectiveElements) [0x00000] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 
  at Vintagestory.Client.NoObf.ShapeTesselator.TesselateShape (Vintagestory.API.Common.Block textureSourceBlock, Vintagestory.API.Common.Shape shape, Vintagestory.API.Client.MeshData& modeldata, Vintagestory.API.MathTools.Vec3f rotation, System.Nullable`1[T] quantityElements, System.String[] selectiveElements) [0x0001e] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 
  at Vintagestory.GameContent.BlockEntityFirepit.getOrCreateMesh (System.String burnstate, System.String contentstate) [0x000b5] in <e160795a538f4d649533661f023b3166>:0 
  at Vintagestory.GameContent.BlockEntityFirepit.OnTesselation (Vintagestory.API.Client.ITerrainMeshPool mesher, Vintagestory.API.Client.ITesselatorAPI tesselator) [0x000a1] in <e160795a538f4d649533661f023b3166>:0 
  at Vintagestory.Client.NoObf.JsonTesselator.Tesselate (Vintagestory.Client.NoObf.TCTCache vars) [0x00145] in <2d6eef6f99694be1be89f6f6b9c7766e>:0 . Block will probably not be rendered as intended.
```

**Comment by darnuria** [comment]
Quick look at the "maybe" triggering code leads here:

[/vssurvivalmod/Systems/Cooking/BlockCookedContainer.cs#L366]
(https://github.com/anegostudios/vssurvivalmod/blob/a2f48afc64c13c881c041b90dc37f8155ecb02c8/Systems/Cooking/BlockCookedContainer.cs#L366)

**Comment by tyronx** [comment]
Thank you, this will be fixed in v1.13
